snapcraft/internal/repo/_deb.py

```diff
@@ -58,15 +58,22 @@
 
 
 class _AptCache:
-    def __init__(self, deb_arch, *, sources_list=None, use_geoip=False):
+    def __init__(self, deb_arch, *, sources_list=None, keyrings=None, use_geoip=False):
         self._deb_arch = deb_arch
         self._sources_list = sources_list
         self._use_geoip = use_geoip
 
+        self._keyrings = list()
+        if keyrings:
+            self._keyrings = keyrings
+
```

Comment:
same lines of code counter proposal:
```python
if not keyrings:
    keyrings = list()
self._keyrings = keyrings
```

or
```python
self._keyrings = keyrings if keyrings else list()
```

Reply:
Sure, fixed.